src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageWithPropertiesExtensions.cs

```diff
@@ -65,14 +66,15 @@ public static class LoggerConfigurationAzureTableStorageWithPropertiesExtensions
 			bool writeInBatches = false,
 			TimeSpan? period = null,
 			int? batchPostingLimit = null,
-			string additionalRowKeyPostfix = null)
+            string additionalRowKeyPostfix = null,
+            Func<LogEvent, string, string> rowKeyFunc = null)
```

Comment:
These are not indented to the same level as the other lines - do you mind fixing this?